src/proper_statem.erl

```diff
@@ -139,6 +139,16 @@
 %%%   function is called both at command generation and command execution time
 %%%   in order to update the model state, therefore the state `S' and the
 %%%   result `Res' can be either symbolic or dynamic.</p></li>
+%%% <li>`all_commands(S::'{@type symbolic_state()}`) ->' {@type [symbolic_call()]}
+%%%   <p>This is an optional callback normally, however it is required for targeted
+%%%   property based testing. Lists all valid symbolic calls, given the current
+%%%   state `S' of the abstract state machine. This list is then filtered based on
+%%%   the preconditions.</p></li>
+%%% <li>`weight(Call::'{@type symbolic_call()}`) -> integer()'
```

Comment:
This reads better as
"When this optional callback is defined and exported, it is used to initialize the weights of commands during targeted property based testing. Otherwise, commands are are given a default initial weight of 1."